docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -173,9 +177,9 @@ To enable Web3 authentication:
 
 ## Multi-factor authentication
 
-Clerk supports multi-factor authentication (MFA), often referred to as two-factor authentication or 2FA. By enabling MFA, you can encourage your users to perform a second verification check during sign-in. By enforcing two different types of verifications, you can drastically improve your user's security. Most websites make this step optional, empowering their users with their own security.
+Clerk supports multi-factor authentication (MFA), also known as two-factor authentication (2FA). Enabling MFA allows you to require users to complete a second verification step during sign-in. This enhances security by enforcing two different types of verification. Many websites offer this as an optional step, giving users control over their own security.
```

Comment:
So enabling it doesn't force MFA for all users. They have to then turn it on for their account.

```suggestion
Clerk supports multi-factor authentication (MFA), also known as two-factor authentication (2FA). If a user enabled MFA for their account, they are required to complete a second verification step during sign-in. This enhances security by enforcing two different types of verification. Many websites offer this as an optional step, giving users control over their own security.
```